src/engraving/libmscore/beam.cpp

```diff
@@ -917,8 +921,9 @@ void Beam::offsetBeamToRemoveCollisions(std::vector<ChordRest*> chordRests, int&
         }
         Chord* chord = toChord(chordRest);
         PointF anchor = chordBeamAnchor(chord);
+
         qreal proportionAlongX = (anchor.x() - startX) / (endX - startX);
-        while (true) {
+        while (true && proportionAlongX) {
```

Comment:
I have some remarks about this line:
- `true && ` could be removed
- the qreal `proportionAlongX` might not become exactly 0, but maybe something like 0.00000001. We often use `RealIsNull` from realutils.h for this. 

Also, it seems that proportionAlongX is not modified inside the loop. Therefore, it might be confusing to use it as a loop condition. Maybe something like this is more suited:
```c++
if (RealIsNull(proportionAlongX)) {
    continue;
}

while (true) {
    ...
```

Reply:
a very good point! I don't know how that slipped in there but that's very bush league of me. It'll be fixed in the next push